src/tokens.ts

```diff
@@ -131,17 +134,24 @@ export class TokenList {
     this.tokens = tokens;
   }
 
+  static async init() {
+    const tokenListResponse = await axios.get(TOKENS_JSON_LIST_URL);
+    const testTokenListResponse = await axios.get(TEXT_TOKENS_JSON_LIST_URL);
```

Comment:
Let's use Promise all here, so we can fetch both in parallel.

Reply:
ok thanks